Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "core-graphics"
 description = "Bindings to Core Graphics for OS X"
 homepage = "https://github.com/servo/core-graphics-rs"
 repository = "https://github.com/servo/core-graphics-rs"
-version = "0.12.4"
+version = "0.12.5"
```

Comment:
Since this crate reexports types from core-foundation, this needs to be bumped to 0.13.0.

Reply:
Good catch. Done! I might need some hints in the other crates what to bump the versions to as well.